code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -28,6 +28,33 @@
 		light.turn_light(null, FALSE, duration, TRUE, TRUE)
 
 
+/datum/action/xeno_action/activable/nightfall/lesser //Less griefy lightfall.
+	name = "Lesser Nightfall"
+	action_icon_state = "nightfall"
+	ability_name = "Lesser Nightfall"
+	mechanics_text = "Shut down all electrical lights for five seconds."
+	cooldown_timer = 5 SECONDS //Nightfall buff
+	plasma_cost = 50 // Cheap
+	/// How far nightfall will have an effect
+	var/range = 8 //Buffed range
+	/// How long till the lights go on again
+	var/duration = 3 SECONDS // 2 Seconds of downtime, enough to make lights consistently flicker
```

Comment:
Comments should probably be adjusted too here.

Reply:
Doing so